pkg/apiserver/controller/queue/queue.go

```diff
@@ -338,26 +338,33 @@ func UpdateQueue(ctx *logger.RequestContext, request *UpdateQueueRequest) (Updat
 
 	// validate fields if not nil, validate namespace at first
 	updateClusterRequired := false
+	resourceUpdated := true
```

Comment:
When does the variable `resourceUpdated` become `false`?

Reply:
resourceUpdated should be false